scrooge-sbt-plugin/src/main/scala/com/twitter/ScroogeSBT.scala

```diff
@@ -179,8 +188,9 @@ object ScroogeSBT extends AutoPlugin {
       streams,
       scroogeThriftSources,
       scroogeThriftOutputFolder,
-      scroogeThriftIncludes
-    ) map { (out, sources, outputDir, inc) =>
+      scroogeThriftIncludes,
+      scroogeGenerated
```

Comment:
How do you ensure that if you change the sbt settings it regenerates the files using the old approach?

Apart from having the behaviour it already had and writing to a different directory or writing out some state I'm not sure how we can achieve what you want?